src/dialog.jsx

```diff
@@ -410,61 +424,113 @@ const DialogInline = React.createClass({
 
 });
 
-
 const Dialog = React.createClass({
 
   propTypes: {
+    /**
+     * **DEPRECATED** The `ref` of the action to focus on when the `Dialog` is displayed.
+     */
     actionFocus: React.PropTypes.string,
-    actions: React.PropTypes.array,
+
+    /**
+     * This prop can be either a JSON object containing the actions to render (This is **DEPRECATED**),
+     * a react elements, or an array of react elements.
+     */
+    actions: React.PropTypes.node,
+
+    /**
+     * If set to true, the height of the `Dialog` will be auto detected. A max height
+     * will be enforced so that the content does not extend beyond the viewport.
+     */
     autoDetectWindowHeight: React.PropTypes.bool,
+
+    /**
+     * If set to true, the body content of the `Dialog` will be scrollable.
+     */
     autoScrollBodyContent: React.PropTypes.bool,
+
+    /**
+     * Overrides the inline-styles of the `Dialog` content's root element under the title.
+     */
     bodyStyle: React.PropTypes.object,
+
+    /**
+     * The contents of the `Dialog`.
+     */
+    children: React.PropTypes.node,
+
+    /**
+     * The `className` to add to the `Dialog` window content container.
+     * This is the `Paper` element that is seen when the `Dialog` is shown.
+     */
```

Comment:
I think we have to make an standard. 

for each xxxStyle we should have xxxClassName and for each style a className, don't you think?